tracing-forest/src/tree/mod.rs

```diff
@@ -105,6 +105,11 @@ pub(crate) struct Shared {
     /// Key-value data.
     #[cfg_attr(feature = "serde", serde(serialize_with = "ser::fields"))]
     pub(crate) fields: FieldSet,
+
```

Comment:
nit: this is only ever set for spans, so it should be a field on `Span`, not `Shared`.